tests/webapp/api/test_push_api.py

```diff
@@ -124,26 +124,26 @@ def test_resultset_list_single_long_revision(webapp, eleven_jobs_stored, test_re
     )
 
 
-def test_resultset_list_single_long_revision_stored_long(webapp, sample_resultset, test_repository):
+def test_push_list_single_long_revision_stored_long(webapp, sample_push, test_repository):
     """
-    test retrieving a resultset list with store long revision, filtered by a single long revision
+    test retrieving a push list with store long revision, filtered by a single long revision
     """
     long_revision = "21fb3eed1b5f3456789012345678901234567890"
 
-    # store a resultset with long revision
-    resultset = copy.deepcopy(sample_resultset[0])
-    resultset["revisions"][0]["revision"] = long_revision
-    store_result_set_data(test_repository, [resultset])
+    # store a push with long revision
+    push = copy.deepcopy(sample_push[0])
+    push["revisions"][0]["revision"] = long_revision
+    store_push_data(test_repository, [push])
 
     resp = webapp.get(
-        reverse("resultset-list", kwargs={"project": test_repository.name}),
+        reverse("push-list", kwargs={"project": test_repository.name}),
         {"revision": long_revision}
     )
     assert resp.status_int == 200
     results = resp.json['results']
     meta = resp.json['meta']
     assert len(results) == 1
-    assert set([rs["revision"] for rs in results]) == {sample_resultset[0]['revision']}
+    assert set([rs["revision"] for rs in results]) == {sample_push[0]['revision']}
```

Comment:
flake8 is failing on this file after changing the name of `rs` to `push`. I think it is because `push` is already defined in the method. I am going to change `rs` to `ph`. At least it has a similar naming convention?
[`push` is defined here](https://github.com/maxchehab/treeherder/blob/master/tests/webapp/api/test_push_api.py#L134)